packages/cli/src/parse-command-line.ts

```diff
@@ -114,6 +120,10 @@ export const parseCommandLine = (
 		Config.Puppeteer.setChromiumHeadlessMode(false);
 	}
 
+	if (parsedCli['every-nth-frame']) {
+		Config.Rendering.setEveryNthFrame(parsedCli['every-nth-frame']);
```

Comment:
We must forbid users setting this property when not rendering a GIF, as it will mess up the audio output